### PR TITLE
translate-/docs/tutorials/clusters/apparmor.md

### PR DESCRIPTION
**Pull request by tanjunchen**
Translate /docs/tutorials/clusters/apparmor.md

### CODE REVIEW

File: content/zh/docs/tutorials/clusters/apparmor.md [hunk 1]
```diff
@@ -0,0 +1,597 @@
+<!-- ---
+reviewers:
+- stclair
+title: AppArmor
+content_template: templates/tutorial
+--- -->
+---
+reviewers:
+- stclair
```
Comment (sftim) [inline content/zh/docs/tutorials/clusters/apparmor.md]: Is this right for zh localization?

**Comment by tanjunchen** [comment]
@tengqm  I Need your Help . :smile:   Thanks

File: content/zh/docs/tutorials/clusters/apparmor.md [hunk 2]
```diff
@@ -0,0 +1,595 @@
...
+<!-- It is recommended to run your application through Docker on a development workstation to generate
+the profiles, but there is nothing preventing running the tools on the Kubernetes node where your
+Pod is running. -->
+建议在开发工作站上通过 Docker 运行应用程序以生成配置文件，但是没有什么可以阻止在运行 Pod 的 Kubernetes 节点上运行工具。
+
+<!-- To debug problems with AppArmor, you can check the system logs to see what, specifically, was
+denied. AppArmor logs verbose messages to `dmesg`, and errors can usually be found in the system
+logs or through `journalctl`. More information is provided in
+[AppArmor failures](https://gitlab.com/apparmor/apparmor/wikis/AppArmor_Failures). -->
+想要调试 AppArmor 的问题，您可以检查系统日志，查看具体拒绝了什么。AppArmor 将详细消息记录到 `dmesg` ，错误通常可以在系统日志中或通过 `journalctl` 找到。更多详细信息见[AppArmor 失败]（https://gitlab.com/apparmor/apparmor/wikis/AppArmor_Failures）。
```
Comment (markthink) [inline content/zh/docs/tutorials/clusters/apparmor.md]: 参考渲染效果：https://deploy-preview-16365--k8s-v1-14.netlify.com/zh/docs/tutorials/clusters/apparmor/

类似文档链接 
[文档链接](http://www.kubernetes.io)  中括号与小括号不要有空格

Comment (tanjunchen) [reply content/zh/docs/tutorials/clusters/apparmor.md]: Thanks!!!

**Pull request merged**